tests/notebook/test_demo_notebook.ipynb

```diff
@@ -2471,6 +2471,7 @@
     }
    ],
    "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
     "pd_flights.describe()"
```

Comment:
Let's update this cell with the Pandas 2 output? (And keep the `IGNORE_OUTPUT` with maybe a comment about it if possible.)

Reply:
Done! The cell now has the pandas 2 output, and this comment will be added:

```suggestion
    "# NBVAL_IGNORE_OUTPUT\n",
    "# Once support for pandas <2 is dropped, this and the line above can be removed\n",
    "pd_flights.describe()"
```